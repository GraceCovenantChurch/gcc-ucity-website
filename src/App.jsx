import React from "react";
import styles from "./App.module.scss";

import BaseRouter from "./pages/BaseRouter";
import Footer from "components/footer/Footer";
import Navigation from "components/navigation/Navigation";
// import { doc } from "prettier";

function App() {
  return (
    <div className={styles.App}>
      <div className={styles.AppContent}>
        <Navigation />
        <BaseRouter />
        <Footer />
      </div>
    </div>
  );
}

export default App;
