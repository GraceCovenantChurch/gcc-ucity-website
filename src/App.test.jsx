import React from "react";
import "./__mocks__/matchMedia";
import { Context as ResponsiveContext } from "react-responsive";
import { render } from "@testing-library/react";

import App from "./App";

describe("App", () => {
  test("matches the snapshot", () => {
    const { container: mobile } = render(
      <ResponsiveContext.Provider value={{ width: 300 }}>
        <App />
      </ResponsiveContext.Provider>
    );
    expect(mobile).toMatchSnapshot();

    const { container: desktop } = render(
      <ResponsiveContext.Provider value={{ width: 1000 }}>
        <App />
      </ResponsiveContext.Provider>
    );
    expect(desktop).toMatchSnapshot();
  });
});
