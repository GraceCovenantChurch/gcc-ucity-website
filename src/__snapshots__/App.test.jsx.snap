// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches the snapshot 1`] = `
<div>
  <div
    class="AppContent"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionSticky MuiAppBar-colorPrimary makeStyles-appBarMobile-7 MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular makeStyles-mobileToolBar-4 MuiToolbar-gutters"
      >
        <div
          class="makeStyles-titleGrow-5"
        >
          <a
            href="/"
          >
            <img
              alt="nav-logo"
              class="makeStyles-menuMobileButton-3"
              src="test-file-stub"
            />
          </a>
        </div>
        <button
          aria-label="menu"
          class="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-9 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </header>
    <div
      class="background"
      style="background-image: url(test-file-stub);"
    >
      <div
        class="contentMobile"
      >
        <div
          class="MuiContainer-root makeStyles-mobileContentContainer-18 MuiContainer-maxWidthLg"
        >
          <p
            class="MuiTypography-root makeStyles-title-22 makeStyles-mobileTitle-23 MuiTypography-body1"
          >
            Raising up kingdom workers
            <br />
            who are transformed by Christ
            <br />
            to influence the world
            <br />
          </p>
          <div
            class="makeStyles-linkContainer-19"
          >
            <a
              class="makeStyles-link-20 undefined"
              href="/beliefs"
            >
              Our beliefs &gt;
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root makeStyles-container-27 MuiContainer-maxWidthLg"
    >
      <h5
        class="MuiTypography-root makeStyles-subtitle-28 MuiTypography-h5"
      >
        SERVICE LOCATION AND TIMES
      </h5>
    </div>
    <div
      class="background"
      style="background-image: url(test-file-stub);"
    >
      <div
        class="contentMobile"
      >
        <div
          class="MuiContainer-root makeStyles-mobileContentContainer-18 MuiContainer-maxWidthLg"
        >
          <p
            class="MuiTypography-root makeStyles-title-22 makeStyles-mobileTitle-23 MuiTypography-body1"
          >
            Family Group
            <br />
          </p>
          <p
            class="MuiTypography-root makeStyles-description-24 makeStyles-mobileDescription-25 MuiTypography-body1"
          >
            You haven't checked out GCC unless
            <br />
            you've checked out our family groups.
            <br />
          </p>
          <div
            class="makeStyles-linkContainer-19"
          >
            <a
              class="makeStyles-link-20 undefined"
              href="/familygroup"
            >
              Find out more &gt;
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <div
        class="MuiContainer-root makeStyles-container-27 MuiContainer-maxWidthLg"
      >
        <h5
          class="MuiTypography-root makeStyles-subtitle-28 MuiTypography-h5"
        >
          UPCOMING EVENTS
        </h5>
        <h5
          class="MuiTypography-root makeStyles-linkContainer-29 MuiTypography-h5"
        >
          <a
            class="makeStyles-learnMore-30"
            href="/events"
          >
            See all events &gt;
          </a>
        </h5>
      </div>
    </div>
    <div
      class="background"
      style="background-image: url(test-file-stub);"
    >
      <div
        class="contentMobile"
      >
        <div
          class="MuiContainer-root makeStyles-mobileContentContainer-32 MuiContainer-maxWidthLg"
        >
          <p
            class="MuiTypography-root makeStyles-title-37 makeStyles-mobileTitle-38 MuiTypography-body1"
          >
            Read With Us
            <br />
          </p>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root makeStyles-container-41 MuiContainer-maxWidthLg"
    >
      <h5
        class="MuiTypography-root makeStyles-subtitle-42 MuiTypography-h5"
      >
        Monthly Memory Verse
      </h5>
    </div>
    <footer
      class="footer"
    >
      <div
        class="MuiContainer-root makeStyles-container-47 MuiContainer-disableGutters MuiContainer-maxWidthLg"
      >
        <div
          class="center"
        >
          <div
            class="MuiGrid-root makeStyles-mobileGrid-48 MuiGrid-container MuiGrid-spacing-xs-5 MuiGrid-justify-content-xs-center"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10"
            >
              <div
                class="MuiPaper-root MuiCard-root makeStyles-card-46 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <h5
                  class="MuiTypography-root makeStyles-title-50 MuiTypography-h5"
                >
                  Service Times
                </h5>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10"
            >
              <div
                class="MuiPaper-root MuiCard-root makeStyles-card-46 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <h5
                  class="MuiTypography-root makeStyles-title-50 MuiTypography-h5"
                >
                  Social
                </h5>
                <div
                  class="center"
                >
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://www.instagram.com/gccphiladelphia/"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7.8 2h8.4C19.4 2 22 4.6 22 7.8v8.4a5.8 5.8 0 0 1-5.8 5.8H7.8C4.6 22 2 19.4 2 16.2V7.8A5.8 5.8 0 0 1 7.8 2m-.2 2A3.6 3.6 0 0 0 4 7.6v8.8C4 18.39 5.61 20 7.6 20h8.8a3.6 3.6 0 0 0 3.6-3.6V7.6C20 5.61 18.39 4 16.4 4H7.6m9.65 1.5a1.25 1.25 0 0 1 1.25 1.25A1.25 1.25 0 0 1 17.25 8 1.25 1.25 0 0 1 16 6.75a1.25 1.25 0 0 1 1.25-1.25M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3z"
                      />
                    </svg>
                  </a>
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://www.facebook.com/gccphiladelphia/"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
                      />
                    </svg>
                  </a>
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://www.youtube.com/user/GraceCovenantChurch"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 15l5.19-3L10 9v6m11.56-7.83c.13.47.22 1.1.28 1.9.07.8.1 1.49.1 2.09L22 12c0 2.19-.16 3.8-.44 4.83-.25.9-.83 1.48-1.73 1.73-.47.13-1.33.22-2.65.28-1.3.07-2.49.1-3.59.1L12 19c-4.19 0-6.8-.16-7.83-.44-.9-.25-1.48-.83-1.73-1.73-.13-.47-.22-1.1-.28-1.9-.07-.8-.1-1.49-.1-2.09L2 12c0-2.19.16-3.8.44-4.83.25-.9.83-1.48 1.73-1.73.47-.13 1.33-.22 2.65-.28 1.3-.07 2.49-.1 3.59-.1L12 5c4.19 0 6.8.16 7.83.44.9.25 1.48.83 1.73 1.73z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10"
            >
              <div
                class="MuiPaper-root MuiCard-root makeStyles-card-46 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <h5
                  class="MuiTypography-root makeStyles-title-50 MuiTypography-h5"
                >
                  Mobile App
                </h5>
                <div
                  class="center"
                >
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://itunes.apple.com/us/app/id651172729?mt=8&uo=4"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M18.71 19.5c-.83 1.24-1.71 2.45-3.05 2.47-1.34.03-1.77-.79-3.29-.79-1.53 0-2 .77-3.27.82-1.31.05-2.3-1.32-3.14-2.53C4.25 17 2.94 12.45 4.7 9.39c.87-1.52 2.43-2.48 4.12-2.51 1.28-.02 2.5.87 3.29.87.78 0 2.26-1.07 3.81-.91.65.03 2.47.26 3.64 1.98-.09.06-2.17 1.28-2.15 3.81.03 3.02 2.65 4.03 2.68 4.04-.03.07-.42 1.44-1.38 2.83M13 3.5c.73-.83 1.94-1.46 2.94-1.5.13 1.17-.34 2.35-1.04 3.19-.69.85-1.83 1.51-2.95 1.42-.15-1.15.41-2.35 1.05-3.11z"
                      />
                    </svg>
                  </a>
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://play.google.com/store/apps/details?id=com.subsplash.thechurchapp.ami&referrer=utm_source%3Dsubsplash%26utm_content%3DeyJoYW5kbGVyIjoiYXBwIiwiYXBwa2V5IjoiNlZaSFZOIn0="
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16 6V4c0-1.11-.89-2-2-2h-4c-1.11 0-2 .89-2 2v2H2v13c0 1.11.89 2 2 2h16c1.11 0 2-.89 2-2V6h-6zm-6-2h4v2h-4V4zM9 18V9l7.5 4L9 18z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="content"
      >
        <p
          class="copyright"
        >
          Copyright © 1996-
          2021
          . Official website of
           
          <a
            class="footerLinks"
            href="/"
          >
            Grace Covenant Church
          </a>
           
          in Philadelphia, PA. All rights reserved.
        </p>
      </div>
    </footer>
  </div>
</div>
`;

exports[`App matches the snapshot 2`] = `
<div>
  <div
    class="AppContent"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionSticky MuiAppBar-colorPrimary makeStyles-appBarMobile-7 MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular makeStyles-mobileToolBar-4 MuiToolbar-gutters"
      >
        <div
          class="makeStyles-titleGrow-5"
        >
          <a
            href="/"
          >
            <img
              alt="nav-logo"
              class="makeStyles-menuMobileButton-3"
              src="test-file-stub"
            />
          </a>
        </div>
        <button
          aria-label="menu"
          class="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-9 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </header>
    <div
      class="background"
      style="background-image: url(test-file-stub);"
    >
      <div
        class="contentMobile"
      >
        <div
          class="MuiContainer-root makeStyles-mobileContentContainer-18 MuiContainer-maxWidthLg"
        >
          <p
            class="MuiTypography-root makeStyles-title-22 makeStyles-mobileTitle-23 MuiTypography-body1"
          >
            Raising up kingdom workers
            <br />
            who are transformed by Christ
            <br />
            to influence the world
            <br />
          </p>
          <div
            class="makeStyles-linkContainer-19"
          >
            <a
              class="makeStyles-link-20 undefined"
              href="/beliefs"
            >
              Our beliefs &gt;
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root makeStyles-container-27 MuiContainer-maxWidthLg"
    >
      <h5
        class="MuiTypography-root makeStyles-subtitle-28 MuiTypography-h5"
      >
        SERVICE LOCATION AND TIMES
      </h5>
    </div>
    <div
      class="background"
      style="background-image: url(test-file-stub);"
    >
      <div
        class="contentMobile"
      >
        <div
          class="MuiContainer-root makeStyles-mobileContentContainer-18 MuiContainer-maxWidthLg"
        >
          <p
            class="MuiTypography-root makeStyles-title-22 makeStyles-mobileTitle-23 MuiTypography-body1"
          >
            Family Group
            <br />
          </p>
          <p
            class="MuiTypography-root makeStyles-description-24 makeStyles-mobileDescription-25 MuiTypography-body1"
          >
            You haven't checked out GCC unless
            <br />
            you've checked out our family groups.
            <br />
          </p>
          <div
            class="makeStyles-linkContainer-19"
          >
            <a
              class="makeStyles-link-20 undefined"
              href="/familygroup"
            >
              Find out more &gt;
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <div
        class="MuiContainer-root makeStyles-container-27 MuiContainer-maxWidthLg"
      >
        <h5
          class="MuiTypography-root makeStyles-subtitle-28 MuiTypography-h5"
        >
          UPCOMING EVENTS
        </h5>
        <h5
          class="MuiTypography-root makeStyles-linkContainer-29 MuiTypography-h5"
        >
          <a
            class="makeStyles-learnMore-30"
            href="/events"
          >
            See all events &gt;
          </a>
        </h5>
      </div>
    </div>
    <div
      class="background"
      style="background-image: url(test-file-stub);"
    >
      <div
        class="contentMobile"
      >
        <div
          class="MuiContainer-root makeStyles-mobileContentContainer-32 MuiContainer-maxWidthLg"
        >
          <p
            class="MuiTypography-root makeStyles-title-37 makeStyles-mobileTitle-38 MuiTypography-body1"
          >
            Read With Us
            <br />
          </p>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root makeStyles-container-41 MuiContainer-maxWidthLg"
    >
      <h5
        class="MuiTypography-root makeStyles-subtitle-42 MuiTypography-h5"
      >
        Monthly Memory Verse
      </h5>
    </div>
    <footer
      class="footer"
    >
      <div
        class="MuiContainer-root makeStyles-container-47 MuiContainer-disableGutters MuiContainer-maxWidthLg"
      >
        <div
          class="center"
        >
          <div
            class="MuiGrid-root makeStyles-mobileGrid-48 MuiGrid-container MuiGrid-spacing-xs-5 MuiGrid-justify-content-xs-center"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10"
            >
              <div
                class="MuiPaper-root MuiCard-root makeStyles-card-46 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <h5
                  class="MuiTypography-root makeStyles-title-50 MuiTypography-h5"
                >
                  Service Times
                </h5>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10"
            >
              <div
                class="MuiPaper-root MuiCard-root makeStyles-card-46 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <h5
                  class="MuiTypography-root makeStyles-title-50 MuiTypography-h5"
                >
                  Social
                </h5>
                <div
                  class="center"
                >
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://www.instagram.com/gccphiladelphia/"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7.8 2h8.4C19.4 2 22 4.6 22 7.8v8.4a5.8 5.8 0 0 1-5.8 5.8H7.8C4.6 22 2 19.4 2 16.2V7.8A5.8 5.8 0 0 1 7.8 2m-.2 2A3.6 3.6 0 0 0 4 7.6v8.8C4 18.39 5.61 20 7.6 20h8.8a3.6 3.6 0 0 0 3.6-3.6V7.6C20 5.61 18.39 4 16.4 4H7.6m9.65 1.5a1.25 1.25 0 0 1 1.25 1.25A1.25 1.25 0 0 1 17.25 8 1.25 1.25 0 0 1 16 6.75a1.25 1.25 0 0 1 1.25-1.25M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3z"
                      />
                    </svg>
                  </a>
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://www.facebook.com/gccphiladelphia/"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
                      />
                    </svg>
                  </a>
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://www.youtube.com/user/GraceCovenantChurch"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 15l5.19-3L10 9v6m11.56-7.83c.13.47.22 1.1.28 1.9.07.8.1 1.49.1 2.09L22 12c0 2.19-.16 3.8-.44 4.83-.25.9-.83 1.48-1.73 1.73-.47.13-1.33.22-2.65.28-1.3.07-2.49.1-3.59.1L12 19c-4.19 0-6.8-.16-7.83-.44-.9-.25-1.48-.83-1.73-1.73-.13-.47-.22-1.1-.28-1.9-.07-.8-.1-1.49-.1-2.09L2 12c0-2.19.16-3.8.44-4.83.25-.9.83-1.48 1.73-1.73.47-.13 1.33-.22 2.65-.28 1.3-.07 2.49-.1 3.59-.1L12 5c4.19 0 6.8.16 7.83.44.9.25 1.48.83 1.73 1.73z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-10"
            >
              <div
                class="MuiPaper-root MuiCard-root makeStyles-card-46 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <h5
                  class="MuiTypography-root makeStyles-title-50 MuiTypography-h5"
                >
                  Mobile App
                </h5>
                <div
                  class="center"
                >
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://itunes.apple.com/us/app/id651172729?mt=8&uo=4"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M18.71 19.5c-.83 1.24-1.71 2.45-3.05 2.47-1.34.03-1.77-.79-3.29-.79-1.53 0-2 .77-3.27.82-1.31.05-2.3-1.32-3.14-2.53C4.25 17 2.94 12.45 4.7 9.39c.87-1.52 2.43-2.48 4.12-2.51 1.28-.02 2.5.87 3.29.87.78 0 2.26-1.07 3.81-.91.65.03 2.47.26 3.64 1.98-.09.06-2.17 1.28-2.15 3.81.03 3.02 2.65 4.03 2.68 4.04-.03.07-.42 1.44-1.38 2.83M13 3.5c.73-.83 1.94-1.46 2.94-1.5.13 1.17-.34 2.35-1.04 3.19-.69.85-1.83 1.51-2.95 1.42-.15-1.15.41-2.35 1.05-3.11z"
                      />
                    </svg>
                  </a>
                  <a
                    class="MuiTypography-root MuiLink-root MuiLink-underlineHover socialLinks MuiTypography-colorPrimary"
                    href="https://play.google.com/store/apps/details?id=com.subsplash.thechurchapp.ami&referrer=utm_source%3Dsubsplash%26utm_content%3DeyJoYW5kbGVyIjoiYXBwIiwiYXBwa2V5IjoiNlZaSFZOIn0="
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16 6V4c0-1.11-.89-2-2-2h-4c-1.11 0-2 .89-2 2v2H2v13c0 1.11.89 2 2 2h16c1.11 0 2-.89 2-2V6h-6zm-6-2h4v2h-4V4zM9 18V9l7.5 4L9 18z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="content"
      >
        <p
          class="copyright"
        >
          Copyright © 1996-
          2021
          . Official website of
           
          <a
            class="footerLinks"
            href="/"
          >
            Grace Covenant Church
          </a>
           
          in Philadelphia, PA. All rights reserved.
        </p>
      </div>
    </footer>
  </div>
</div>
`;
